app/components/HeaderNotiIcon.tsx

```diff
@@ -184,8 +184,19 @@ const HeaderNotiIcon = (props: any): any => {
               })
             }
             if (currentChildNotis.vcnotis) {
+              // 
               currentChildNotis.vcnotis.forEach((item: any) => {
+                console.log(item,"..currentChildNotis");
+                if(item.title=="vcNoti1"){
+                const vcNotisExists= getVaccinesForPeriodCount(allVaccineData, item.growth_period);
```

Comment:
Please fix the indentation, spacing and format issues all over the file

Reply:
Noted and done